docs/changelog/1439.md

```diff
@@ -0,0 +1 @@
+- Added `mapping:axial-geometric-multiscale` and `mapping:radial-geometric-multiscale` mapping types.
```

Comment:
Maybe adding information what isn't supported may be useful here. Also you could remove `mapping:` as it is redundant with `mapping type`.